enhancements/network/external-lb-vips.md

```diff
@@ -0,0 +1,268 @@
+---
+title: external-lb-vips
+authors:
+  - "@emilienm"
+reviewers: 
+  - "@cybertron for SME review on on-premise networking"
+  - "@jcpowermac for SME review on installer"
+  - "@patrickdillon for SME review and approval on installer"
+  - "@JoelSpeed for SME review on API and general feedback"
+  - "@deads2k for SME review on API and general feedback"
+approvers:
+  - "@cybertron"
+  - "@patrickdillon"
+  - "@JoelSpeed"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-01-23
+last-updated: 2023-02-01
+tracking-link:
+  - https://issues.redhat.com/browse/OCPBU-156
+  - https://issues.redhat.com/browse/OSASINFRA-3069
+see-also:
+  - "/enhancements/on-prem-service-loadbalancers.md"
+replaces:
+  - https://github.com/openshift/enhancements/pull/1281
+---
+
+# External loadbalancer VIPs
+
+## Summary
+
+Originally the on-premise IPI architecture was designed to deploy an internal
+loadbalancer based on HAproxy, Keepalived.
+When customers needed more flexibility for the control-plane VIPs management, they would
+deploy with the UPI method.
+
+Also, customers have the possibility to use another loadbalancer ([example with OpenStack](https://docs.openshift.com/container-platform/4.12/networking/load-balancing-openstack.html#nw-osp-configuring-external-load-balancer_load-balancing-openstack))
+for the these VIPs but OpenShift would still deploy Keepalived and HAproxy in the
```

Comment:
typo: "the these"

but also, which VIPs?

Reply:
L34, control plane VIPs.
Thanks for the typo.